GVFS/GVFS.Service/Mac/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using GVFS.Common;
+using GVFS.Common.Tracing;
+using GVFS.PlatformLoader;
+using System;
+
+namespace GVFS.Service
+{
+    public static class Program

```

Comment:
To confirm, is this the machine-wide launchd service, or will this be the per-user service?